tests/test_json_validation.py

```diff
@@ -20,6 +21,22 @@ def test_all_schemas_are_valid():
             yield check_schema_file, file_path
 
 
+def test_updater_json_validates_correctly():
+    invalid_updater_no_reason = {'updated_by': 'this'}
+    invalid_updater_no_username = {'update_reason': 'this'}
+    invalid_updater_no_fields = {'invalid': 'this'}
+    valid_updater = {'updated_by': 'this', 'update_reason': 'hi'}
+
+    assert_equal(validates_against_schema(
+        UPDATER_SCHEMA, invalid_updater_no_reason), False)
+    assert_equal(validates_against_schema(
+        UPDATER_SCHEMA, invalid_updater_no_username), False)
+    assert_equal(validates_against_schema(
+        UPDATER_SCHEMA, invalid_updater_no_fields), False)
+    assert_equal(validates_against_schema(
+        UPDATER_SCHEMA, valid_updater), True)
```

Comment:
It would be nice to stick to using test generators (instead of having a series of asserts) for things like this. That way each check will pass/fail independently.